workspaceitem-data-cclicense.md

```diff
@@ -0,0 +1,74 @@
+# WorkspaceItem data of CC License sectionType
+[Back to the definition of the workspaceitems endpoint](workspaceitems.md)
+
+The section data represents the data about the CC license
+
+```json
+{
+  	"granted": true,
+  	"uri": "http://creativecommons.org/licenses/by-nc-sa/3.0/us/",
+  	"rights": "Attribution-NonCommercial-ShareAlike 3.0 United States",
+	"file":  
+  	 	{
+          "uuid": "1ce6db0e-662f-4a13-ba87-c371ad664b14",
+          "name": "license_rdf",
+          "handle": null,
+          "metadata": {
+              "dc.title": [
+                {
+                  "value": "license_rdf",
+                  "language": null,
+                  "authority": null,
+                  "confidence": -1,
+                  "place": 0
+                }
+              ]
+          },
+          "sizeBytes": 1012,
+          "checkSum": {
+            "checkSumAlgorithm": "MD5",
+            "value": "59a4cfd819ce67ba7252efc52d1eb99b"
+          },
+          "sequenceId": 1,
+          "type": "bitstream"
+        }
+}
+```
+
+The following attributes can be defined:
+* granted: true if a CC License has been specified
+* uri: the URI of the CC License, typically stored in dc.rights.uri, can be null
+* rights: the rights name of the CC License, typically stored in dc.rights, can be null
+* file: the bitstream containing the license, can be null
+
+## Patch operations
+The PATCH method expects a JSON body according to the [JSON Patch specification RFC6902](https://tools.ietf.org/html/rfc6902)
+
+Each successful Patch operation will return a HTTP 200 CODE with the new workspaceitem as body
+
+### Add
+To accept a CC license the client must send a JSON Patch ADD operation of the uri retrieved from the [Search CC License](submissioncclicenses.md#search-cc-license)
+
+```json
+[
+  {
+    "op": "add",
+    "path": "/sections/<:name-of-the-form>/uri",
+    "value": "http://creativecommons.org/licenses/by-nc-sa/3.0/us/"
+  }
+]
+```
+
+The dc.rights, dc.rights.uri and RDF bitstream will be retrieved from e.g. https://api.creativecommons.org/rest/1.5/details?license-uri=http://creativecommons.org/licenses/by-nc-sa/3.0/ 
+
+Please note that according to the [JSON Patch specification RFC6902](https://tools.ietf.org/html/rfc6902) a subsequent add operation on the granted will have the effect to replace the previous granted license with a new one. 
+In this case a new CC license will be added to the item and the previous license deleted.
+
+It is also possible to send a PATH add operation using *false* as value to reject / remove a license.
```

Comment:
no please, there is no need for this special treatment

Reply:
It seems I just copied this from https://github.com/DSpace/Rest7Contract/blob/master/workspaceitem-data-license.md#add
I'll remove it in both places since it indeed doesn't make sense

Reply:
in the workspaceitem-data-license.md is required because the granted attribute is the one that trigger the acceptance or less of the license. The angular app cannot provide the licenseurl in such case (as it will be the result of the bitstream license creation in the backend) nor provide an acceptanceDate as this will pose a security issue (the timestamp is detected on the server side). So please rollback the change on the workspace-data-license contract

Reply:
I've moved this change to https://github.com/DSpace/Rest7Contract/pull/118 so the fix can still be applied, but doesn't hold back this PR
The PATCH REMOVE should be used there